shared/common-adapters/icon.native.js

```diff
@@ -109,28 +104,26 @@ class Icon extends React.PureComponent<Props> {
 
     if (iconMeta[iconType].isFont) {
       const code = String.fromCharCode(iconMeta[iconType].charCode || 0)
-
+      // glamorous.text color just doesn't work? Put color
+      // back into the style here
+      if (props.color) {
```

Comment:
i'd like to understand what's happening here. can we debug this a bit? i assume this is running?https://github.com/keybase/client/blob/343518d75fe08c28cc3509e1b5537c8017ed2cc9/shared/common-adapters/icon.native.js#L39

Reply:
hmm when I break in the debugger in ios simulator it looks like props here is an array of styles, not a single style object, at least at some point after I click the people tab:
<img width="876" alt="screen shot 2018-04-20 at 4 26 49 pm" src="https://user-images.githubusercontent.com/862397/39077577-e0fc267a-44b7-11e8-9389-14b95b10f884.png">

Reply:
ok nvm I got this one - "props" here is what's passed into the glamorous text object, not the overall icon